src/libraries/System.Net.Primitives/src/System/Net/SocketException.cs

```diff
@@ -30,12 +30,23 @@ public SocketException(int errorCode) : this((SocketError)errorCode)
             // but that's the least bad option right now.
         }
 
+        /// <summary>Creates a new instance of the <see cref='System.Net.Sockets.SocketException'/> class with the specified error code and optional message.</summary>
```

Comment:
The standard wording for constructors is "Initializes".

```suggestion
        /// <summary>Initializes a new instance of the <see cref='System.Net.Sockets.SocketException'/> class with the specified error code and optional message.</summary>
```